### PR TITLE
Add upsell banner for Eventbrite Tickets on the EA form.

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/107108

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [import-form.php:303](https://github.com/moderntribe/the-events-calendar/blob/bf0178a8932daaf3f0f9290bb2ac442777e59d8e/src/admin-views/aggregator/tabs/import-form.php#L303)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for *M18.10*, but this was pull requested against *release/M18.09*.

File: src/Tribe/Aggregator/Tabs/New.php [hunk 1]
```diff
@@ -559,6 +559,45 @@ public function maybe_display_aggregator_upsell() {
...
+			return;
+		}
+
+		if ( class_exists( 'Tribe__Events__Tickets__Eventbrite__Main' ) ) {
+			return;
+		}
+
+		ob_start();
+		?>
+		<div class="tribe-dependent tribe-banner tribe-banner-eventbrite-tickets" data-depends="#tribe-ea-field-origin" data-condition="eventbrite">
```
Comment (bordoni) [inline src/Tribe/Aggregator/Tabs/New.php]: Why not shove the HTML into a view?

Comment (juanfra) [reply src/Tribe/Aggregator/Tabs/New.php]: We can do that. I tried to use the same logic as the other upsell banner to keep uniformity.

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [import-form.php:303](https://github.com/moderntribe/the-events-calendar/blob/fd322585d681f57850d5ef5e349f66cd529d436d/src/admin-views/aggregator/tabs/import-form.php#L303)

> via jenkins/codesniffer-phpcs

File: src/resources/images/aggregator/eventbrite-tickets.svg [hunk 2]
```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (mitogh) [inline src/resources/images/aggregator/eventbrite-tickets.svg]: You might want to save some bytes here by compressing the SVG, a handy tool:

- https://jakearchibald.github.io/svgomg/

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [import-form.php:303](https://github.com/moderntribe/the-events-calendar/blob/5bad2b0cf827438d21791c903960c112486523a9/src/admin-views/aggregator/tabs/import-form.php#L303)

> via jenkins/codesniffer-phpcs

**Comment by mitogh** [comment]
Sorry! Done @juanfra 🌮

**Comment by juanfra** [comment]
Thanks a ton! 🌮

**Pull request merged**